src/caffeine/prif_queries_s.f90

```diff
@@ -26,8 +26,8 @@
     end if
   end procedure
 
-  module procedure prif_local_data_size
-    call unimplemented("prif_local_data_size")
+  module procedure prif_size_bytes
+    call unimplemented("prif_size_bytes")
```

Comment:
Slightly off-topic, but FWIW I think the implementation here is trivially:
```suggestion
    data_size = coarray_handle%info%coarray_size
```

Reply:
Thanks @bonachea. I have added issue #113 so this can not be forgot but be addressed in a different PR.